u3-views/app.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+const expHbs = require('express-handlebars')
+const { courseDetails, courseUnits, courseStudent } = require('./features/courseController')
+const { index } = require('./features/indexController')
+const { videoplayer, DBTube } = require('./features/videoController')
```

Comment:
ฉันแนะนำให้คุณใช้ `lowerCamelCase` เพื่อตั้งชื่อตัวแปรของคุณ ทุกตัวแปรควรเริ่มต้นด้วยอักษรตัวพิมพ์เล็ก และใช้เฉพาะตัวพิมพ์ใหญ่เพื่อเริ่มคำใหม่:
```js
videoPlayer
dbTube
```

I recommend that you use `lowerCamelCase` to name your variables. Every variable should start with a lowercase letter, and only use uppercase letters to start a new word:
```js
videoPlayer
dbTube
```

นอกจากนี้ DBTube ยังเป็นชื่อของเว็บไซต์อีกด้วย ฉันแนะนำให้คุณตั้งชื่อตัวควบคุมของคุณเช่น "videoHome" เพื่อให้ชัดเจนยิ่งขึ้นว่าการตอบสนองจะเป็นอย่างไร / Also, DBTube is the name of the entire videos website. I recommend a name like `videoHome` the makes it clearer what the response will be.